src/requirements/data/majors/bio.ts

```diff
@@ -113,7 +112,6 @@ const bioRequirements: readonly CollegeOrMajorRequirement[] = [
     fulfilledBy: 'courses',
     perSlotMinCount: [1, 1],
   },
-  // TODO: temp fix for reqs w options and diff minCount
```

Comment:
can you elaborate

Reply:
oh this was an earlier comment I had but perSlotMinCount fixed this problem!